### DISCUSSION TITLE
spaCy inference without pytorch dependency?

### DISCUSSION THREAD

**Discussion by rbracco** [explosion/spaCy#10271] [Help: Installation] [answered]
Hi,

Is there any way to use spaCy pretrained models for inference (specifically on lemmatization/part-of-speech tagging) without pytorch dependencies (e.g. Onnx models or torchscript)? I have a containerized app using a docker image, and torch + other dependencies cause it to be 1.8GB and if I could remove pytorch it'd be ~200MB. 

I tried searching discussions and issues for jit, torchscript, onnx, and pytorch dependency and found nothing. Thank you!

**Comment by polm** [comment, marked as answer]
Torch is only required if you're using spaCy transformers. If you use the sm/md/lg models you don't need it.
